packages/ui-core/src/utils/convert.tsx

```diff
@@ -1,51 +1,92 @@
 import React from 'react';
 import convert from 'htmr';
 
+type AttrName = string;
+
+type PropName = string;
+
+type ConfigItemPropsType = string[] | Record<AttrName, PropName>;
```

Comment:
Можно не выносить в переменные, если больше нигде этот тип не нужно использовать
```suggestion
type ConfigItemPropsType = string[] | Record<string, string>;
```